config/crd/bases/anywhere.eks.amazonaws.com_cloudstackdeploymentconfigs.yaml

```diff
@@ -0,0 +1,78 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.1
+  creationTimestamp: null
+  name: cloudstackdeploymentconfigs.anywhere.eks.amazonaws.com
+spec:
+  group: anywhere.eks.amazonaws.com
+  names:
+    kind: CloudStackDeploymentConfig
```

Comment:
What is a `CloudStackDeploymentConfig`? Can we make this `CloudStackDatacenterConfig` to make it consistent with the other providers?

Reply:
Also maybe need to go through a design doc of what fields we are adding in terms of custom resources

Reply:
The reason we decided to go with CloudStackDeploymentConfig originally was that there is not necessarily a 1:1 mapping between a datacenter and a particular CloudStack deployment/endpoint. A CloudStack deployment may manage multiple datacenters, aka CloudStack "zones", and when we are executing API's, it is against a particular CloudStack deployment.

The fields we are adding align pretty closely with what is present in CAPC (https://github.com/aws/cluster-api-provider-cloudstack) but I'll have to look for a doc that outlines each one specifically.